guides/hosting/infrastructure/reverse-http-cache.md

```diff
@@ -31,20 +31,27 @@ This setup is compatible from Shopware version 6.4.
 First, we need to activate the reverse proxy support in Shopware. To enable it, we need to create a new file in `config/packages/storefront.yaml`:
 
 ```yaml
-storefront:
-    reverse_proxy:
-        enabled: true
-        ban_method: "BAN"
-        # This needs to point to your varnish hosts
-        hosts: [ "http://varnish" ]
-        # Max parallel invalidations at same time for a single worker
-        max_parallel_invalidations: 3
-        # Redis Storage for the HTTP cache tags
-        redis_url: "redis://redis"
+shopware:
+    http_cache:
+        reverse_proxy:
+            enabled: true
+            ban_method: "BAN"
+            # This needs to point to your varnish hosts
+            hosts: [ "http://varnish" ]
+            # Max parallel invalidations at same time for a single worker
+            max_parallel_invalidations: 3
+            # Redis Storage for the HTTP cache tags
+            redis_url: "redis://redis"
 ```
 
 Also set `SHOPWARE_HTTP_CACHE_ENABLED=1` in your `.env` file.
 
+::: info
+The configuration key changed from `storefront.reverse_proxy` up to Shopware 6.5.x to `shopware.http_cache.reverse_proxy` starting with Shopware 6.6.0.0.
+So you will need to adjust your config while upgrading.
+If you look for the old documentation and examples, you can find it [here](https://developer.shopware.com/docs/v6.5/guides/hosting/infrastructure/reverse-http-cache.html)
```

Comment:
right now the link is broken, but it should be the correct one when this branch is the dedault